anonlinkclient/utils.py

```diff
@@ -124,6 +124,11 @@ def generate_candidate_blocks_from_csv(input_f: TextIO,
 
     # step3 - get config meta data
     result['config'] = blocking_config
+
+    # step4 - add CLK counts and blocking statistics to metadata
+    result['clk_count'] = len(encoding_to_blocks_map)
```

Comment:
just putting 'clk_count' in the top level of the output is ambiguous. Which CLKs are counted here?

Reply:
CLKs that belongs to any block